There is an error when compiling the firmware for Raspberry Pi Pico, but compiling ARDUINO_NANO_RP2040_CONNECT is possible.

**During compilation, an error is initially reported that PICO is not found. After modifying it to RPI_PICO and recompiling, the following error occurs:**

-------------------------------------
Install the project...
-- Install configuration: "Release"
make[5]: Leaving directory '/Source/openmv/openmv/src/build/rp2/_deps/picotool-build'
[  3%] Performing test step for 'picotoolBuild'
picotool v2.0.0 (Linux, GNU-13.2.0, Release)
[  3%] Completed 'picotoolBuild'
make[4]: Leaving directory '/Source/openmv/openmv/src/build/rp2'
[  3%] Built target picotoolBuild
make[3]: Leaving directory '/Source/openmv/openmv/src/build/rp2'
make[2]: *** [Makefile:91: all] Error 2
make[2]: Leaving directory '/Source/openmv/openmv/src/build/rp2'
-e See https://github.com/micropython/micropython/wiki/Build-Troubleshooting
make[1]: *** [Makefile:59: all] Error 1
make[1]: Leaving directory '/Source/openmv/openmv/src/micropython/ports/rp2'
make: *** [/Source/openmv/openmv/src/omv/ports/rp2/omv_portconfig.mk:37: firmware] Error 2
make: Leaving directory '/Source/openmv/openmv/src
-----------------------------------

Could you please tell me how I should troubleshoot the error?

=============================
Here is the screenshot of my successful compilation of ARDUINO_NANO_RP2040_CONNECT:![image](https://github.com/user-attachments/assets/e08d37da-704a-4544-aa8b-d890488c1c1c)

--- a/src/omv/ports/rp2/main.c
+++ b/src/omv/ports/rp2/main.c
@@ -45,6 +45,7 @@
 #include "mpbthciport.h"
 #include "mpnetworkport.h"
 #include "genhdr/mpversion.h"
+
 #include "pico/stdlib.h"
 #include "pico/binary_info.h"
 #include "pico/unique_id.h"
@@ -53,6 +54,8 @@
 #include "hardware/regs/intctrl.h"
 #include "hardware/structs/rosc.h"
 #include "pico/bootrom.h"
+#include "pico/aon_timer.h"
+#include "shared/timeutils/timeutils.h"
 
 #include "omv_boardconfig.h"
 #include "framebuffer.h"
@@ -107,6 +110,19 @@ void pico_reset_to_bootloader(size_t n_args, const void *args_in) {
 int main(int argc, char **argv) {
     bool first_soft_reset = true;
 
+    // This is a tickless port, interrupts should always trigger SEV.
+    #if PICO_ARM
+    SCB->SCR |= SCB_SCR_SEVONPEND_Msk;
+    #endif
+
+    soft_timer_init();
+
+    // Set the MCU frequency and as a side effect the peripheral clock to 48 MHz.
+    set_sys_clock_khz(SYS_CLK_KHZ, false);
+
+    // Hook for setting up anything that needs to be super early in the bootup process.
+    MICROPY_BOARD_STARTUP();
+
     #if MICROPY_HW_ENABLE_UART_REPL
     bi_decl(bi_program_feature("UART REPL"))
     setup_default_uart();
@@ -127,17 +143,9 @@ int main(int argc, char **argv) {
     #endif
 
     // Start and initialise the RTC
-    datetime_t t = {
-        .year = 2021,
-        .month = 1,
-        .day = 1,
-        .dotw = 4, // 0 is Monday, so 4 is Friday
-        .hour = 0,
-        .min = 0,
-        .sec = 0,
-    };
-    rtc_init();
-    rtc_set_datetime(&t);
+    struct timespec ts = { 0, 0 };
+    ts.tv_sec = timeutils_seconds_since_epoch(2021, 1, 1, 0, 0, 0);
+    aon_timer_start(&ts);
     mp_hal_time_ns_set_from_rtc();
 
     // Set board unique ID from flash for USB debugging.

